app/models/pillar.rb

```diff
@@ -36,4 +36,23 @@ class Pillar < ApplicationRecord
   validates :value, presence: true
 
   scope :global, -> { where minion_id: nil }
+
+  class << self
+    def value(pillar:)
+      pillar = Pillar.find_by pillar: all_pillars[pillar]
+      return pillar.value if pillar
+      nil
```

Comment:
[minor] these 2 lines can be one: `pillar && pillar.value` or even `pillar.try(:value)`